docs/usage/files.md

```diff
@@ -689,6 +696,25 @@ print('Applied metadata in instance ID {0}'.format(applied_metadata['$id']))
 [metadata]: https://box-python-sdk.readthedocs.io/en/latest/boxsdk.object.html#boxsdk.object.item.Item.metadata
 [metadata_create]: https://box-python-sdk.readthedocs.io/en/latest/boxsdk.object.html#boxsdk.object.metadata.Metadata.create
 
+Set Metadata
```

Comment:
As I mentioned in the Java SDK pull request, I think we should condense the existing documentation for "create" and "update" metadata into this section as additional information.  This new method should be the default for most developers, and it would be helpful if the table of contents above didn't have three sections for them to pick from when trying to figure out how to set metadata!